pom.xml

```diff
@@ -65,7 +65,7 @@
     <version.trino>359</version.trino>
     <version.spring.boot>2.3.8.RELEASE</version.spring.boot>
     <version.guava>30.0-jre</version.guava>
-    <version.nimbus>8.6</version.nimbus>
+    <version.nimbus>9.22</version.nimbus>
```

Comment:
I was experiencing exceptions at startup because of a missing method in this library. This version (9.22) matches what the current version of Spring Security depends on. Not sure why it's working for anyone else...